plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergFileWriterFactory.java

```diff
@@ -84,6 +94,7 @@
     private final FileFormatDataSourceStats readStats;
     private final OrcWriterStats orcWriterStats = new OrcWriterStats();
     private final OrcWriterOptions orcWriterOptions;
+    private static final Splitter COLUMN_NAMES_SPLITTER = Splitter.on(',').trimResults().omitEmptyStrings();
```

Comment:
Move up next to the other `static final` field

Reply:
Done.